hcl/hcl.go

```diff
@@ -813,7 +823,7 @@ func validateGenerateFileBlock(block *ast.Block) error {
 // as is (original expression form, no evaluation).
 //
 // Returns an error if the evaluation fails.
-func CopyBody(target *hclwrite.Body, src *hclsyntax.Body, eval Evaluator) error {
+func CopyBody(dest *hclwrite.Body, src *hclsyntax.Body, eval Evaluator) error {
```

Comment:
While I implemented this I realized that maybe CopyBody should be on genhcl. The logic here is by now pretty genhcl specific, it is not a generic/plain HCL body copying anymore, thanks to all the tm_dynamic logic. But I'm not sure so I didn't moved anything yet, wondering :thinking: